src/control/server/main.go

```diff
@@ -27,16 +27,22 @@ import (
 	"fmt"
 	"net"
 	"os"
+	"os/signal"
+	"syscall"
 
 	mgmtpb "github.com/daos-stack/daos/src/control/common/proto/mgmt"
+	"github.com/daos-stack/daos/src/control/ioserver"
 	"github.com/daos-stack/daos/src/control/log"
 	"github.com/daos-stack/daos/src/control/security/acl"
 	flags "github.com/jessevdk/go-flags"
+	"github.com/pkg/errors"
+	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 )
 
 func main() {
-	if serverMain() != nil {
+	if err := serverMain(); err != nil {
+		log.Errorf("%+v\n", err)
```

Comment:
we had discussions about having a catchall log here, @liw can you remember what consensus was?

Reply:
As it is currently, there are some errors which are not logged and not handled at all. I believe @dpquigl was seeing that while developing his certificate feature and that's why he added a fmt.Printf() to ensure that he saw the error.